src/main/java/com/adobe/pdfservices/operation/samples/exportpdftoimages/ExportPDFToJPEG.java

```diff
@@ -30,10 +30,10 @@
  * <p>
  * Refer to README.md for instructions on how to run the samples.
  */
-public class ExportPDFToJPEGList {
+public class ExportPDFToJPEG {
 
     // Initialize the logger.
-    private static final Logger LOGGER = LoggerFactory.getLogger(ExportPDFToJPEGList.class);
+    private static final Logger LOGGER = LoggerFactory.getLogger(ExportPDFToJPEG.class);
```

Comment:
We can change the output naming for this class from exportPDFToImagesOutput_ to exportPDFToJPEGOutput_